composer.json

```diff
@@ -15,11 +15,16 @@
 		"stecman/symfony-console-completion": "^0.6.1"
 	},
 	"require-dev": {
-		"aik099/coding-standard": "dev-master"
+		"aik099/coding-standard": "dev-master",
+		"phpunit/phpunit": "^4.8 || ^5.7 || ^6.5 || ^7.0"
 	},
 	"autoload": {
 		"psr-4": {
-			"ConsoleHelpers\\ConsoleKit\\": "src/ConsoleKit/",
+			"ConsoleHelpers\\ConsoleKit\\": "src/ConsoleKit/"
+		}
+	},
+	"autoload-dev": {
```

Comment:
Nice catch.

Please update `composer.lock` file to reflect that. Keep in mind, that you need to use PHP 5.3 to do that, because otherwise you're likely will get Fatal Error during test run on PHP 5.3.

Reply:
According to the `composer.json`, it looks like the package requires `php-5.4` version at least.

And I think we don't have the test for `php-5.3` version in Travis CI build :).

Reply:
> According to the `composer.json`, it looks like the package requires `php-5.4` version at least.

My mistake. I do have some projects, that use PHP 5.3 and I thought this one was do that too.

> And I think we don't have the test for `php-5.3` version in Travis CI build :).

It does, but you need to specify different virtual machine image to do this: https://docs.travis-ci.com/user/reference/trusty#php-images